examples/kcp-certs/run.bash

```
@@ -0,0 +1,270 @@
+#!/usr/bin/env bash
+
+# cd into repo root
+example_dir="$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] example_dir appears unused. Verify use (or export if used externally).

(SC2034)